avocado/core/result.py

```diff
@@ -165,6 +166,7 @@ def end_test(self, state):
         """
         self.tests_run += 1
```

Comment:
This is indeed duplicate information, but public.  It's used, for example, in the TAP plugin.  The `Result` class will receive major changes and cleanups, so my idea is to leave that kind of change to the next stage.  How does it sound?